libs/dh/metering-point/feature-overview/src/lib/dh-metering-point-details.component.ts

```diff
@@ -104,35 +106,35 @@ import { MeteringPointDetails } from './types';
           <watt-description-list variant="stack" [itemSeparators]="false">
             <watt-description-list-item
               [label]="t('meteringPointType')"
-              [value]="null | dhEmDashFallback"
+              [value]="meteringPoint()?.type | dhEmDashFallback"
             />
             <watt-description-list-item
               [label]="t('meteringPointKind')"
               [value]="null | dhEmDashFallback"
             />
             <watt-description-list-item
               [label]="t('meteringPointNumber')"
-              [value]="null | dhEmDashFallback"
+              [value]="meteringPoint()?.meteringPointId | dhEmDashFallback"
             />
             <watt-description-list-item
               [label]="t('settlementMethod')"
               [value]="null | dhEmDashFallback"
             />
             <watt-description-list-item
               [label]="t('electricalHeating')"
-              [value]="null | dhEmDashFallback"
+              [value]="hasElectricalHeating() ? t('yes') : (t('no') | dhEmDashFallback)"
```

Comment:
issue: "yes" and "no" translations are missing. Unless it's intended to look them up in the global scope. In this case the pipe should be used.

issue: the `dhEmDashFallback` will not work as intended. The "No" translation will always be shown (when the translation path is fixed)